apscheduler/util.py

```diff
@@ -384,3 +384,9 @@ def check_callable_args(func, args, kwargs):
         raise ValueError(
             'The target callable does not accept the following keyword arguments: %s' %
             ', '.join(unmatched_kwargs))
+
+
+def iscoroutinefunction_partial(f):
+    while isinstance(f, partial):
+        f = f.func
+    return iscoroutinefunction(f)
```

Comment:
Add a blank line after the `while` block.

Reply:
These will no longer abut.